From: Jeroen Dekkers <jeroen@dekkers.ch>
Date: Wed, 7 Dec 2011 21:11:45 +0100
Subject: ckeditor-nospellcheck

---
 UI/WebServerResources/ckeditor/config.js |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/UI/WebServerResources/ckeditor/config.js b/UI/WebServerResources/ckeditor/config.js
index c54877e..48674e9 100644
--- a/UI/WebServerResources/ckeditor/config.js
+++ b/UI/WebServerResources/ckeditor/config.js
@@ -7,7 +7,7 @@ CKEDITOR.editorConfig = function( config )
 {
     config.uiColor = '#D4D0C8';
     config.skin = 'kama';
-    config.removePlugins = "elementspath,maximize,resize";
+    config.removePlugins = "elementspath,maximize,resize,scayt";
     config.scayt_autoStartup = false;
     config.enterMode = CKEDITOR.ENTER_BR;
 };
