From: Jeroen Dekkers <jeroen@dekkers.ch>
Date: Sat, 30 Jun 2012 02:49:20 +0200
Subject: Compile daemon as PIE

---
 Main/GNUmakefile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Main/GNUmakefile b/Main/GNUmakefile
index 4b61d72..835ef23 100644
--- a/Main/GNUmakefile
+++ b/Main/GNUmakefile
@@ -4,9 +4,10 @@ include ../config.make
 include $(GNUSTEP_MAKEFILES)/common.make
 include ../Version
 
+ADDITIONAL_OBJCFLAGS += -fPIE
 ADDITIONAL_INCLUDE_DIRS += 
 ADDITIONAL_LIB_DIRS += -L../SOPE/GDLContentStore/obj/
-ADDITIONAL_LDFLAGS += -Wl,--no-as-needed -Wl,--rpath,/usr/lib/sogo
+ADDITIONAL_LDFLAGS += -Wl,--no-as-needed -Wl,--rpath,/usr/lib/sogo -fPIE -pie
 
 SOGOD     = sogod
 TOOL_NAME = $(SOGOD)
